### ISSUE TITLE
Ask for New Release

### ISSUE THREAD

**Issue by nitrogenice** [i-schuetz/PieCharts#5]
Hi, our team was hoping to use PieCharts through Cocoapods, with my pull request merged in the project. The current release does not have options for using PieChart without Xibs, while our team's internal policy is to not use Xibs.

Could you do us a favor of releasing a new version of PieCharts?

Thanks!

**Comment by i-schuetz** [comment]
Done (released 0.0.4).

Note that in these cases you also can use master directly:

```ruby
pod 'SwiftCharts', :git => 'https://github.com/i-schuetz/SwiftCharts.git'
```
Anyway this was an important fix and a new release made totally sense, thanks!

**Comment by nitrogenice** [comment]
Oh, never new that. Thanks for the work!
